sh_notification.lua is broken

Installed evolve and get this error when I try "!notice"

[ERROR] addons/evolve/lua/ev_plugins/sh_notification.lua:41: attempt to call method 'AddNotify' (a nil value)
1. func - addons/evolve/lua/ev_plugins/sh_notification.lua:41
   1. unknown - lua/includes/modules/net.lua:31

--- a/lua/ev_plugins/sh_notification.lua
+++ b/lua/ev_plugins/sh_notification.lua
@@ -38,7 +38,7 @@ if ( CLIENT ) then
 		local time = net.ReadUInt(8)
 		local msg = net.ReadString()
 		
-		GAMEMODE:AddNotify( msg, NOTIFY_GENERIC, time )
+		notification.AddLegacy( msg, NOTIFY_GENERIC, time )
 		surface.PlaySound( "ambient/water/drip" .. math.random( 1, 4 ) .. ".wav" )
 	end )
 end

